azure-pipelines.yml

```diff
@@ -171,7 +171,7 @@ jobs:
           eq(variables['Build.SourceBranch'], 'refs/heads/master'))
 
   - script: |
-      cd doc/build/html
+      cd to_deploy
```

Comment:
Where is `to_deploy` defined?

Reply:
It's created here, is the directory where the web and docs is copied together: https://github.com/pandas-dev/pandas/blob/master/azure-pipelines.yml#L136

Reply:
Sorry, I apparently wasn't on master (so my grep didn't find it).